lib/generators/boring/brakeman/install/install_generator.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Boring
+  module Brakeman
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Brakeman to the application'
+      source_root File.expand_path('templates', __dir__)
+
+      RUBY_VERSION_FILE = '.ruby-version'
+
+      DEFAULT_RUBY_VERSION = '.ruby-version'
+
+      class_option :skip_config, type: :boolean, aliases: '-c',
+                                                desc: 'Skip adding Brakeman configuration'
+
+      class_option :skip_github, type: :boolean, aliases: '-gh',
+                                                desc: 'Skip adding GitHub Actions configuration'
+
+      class_option :skip_gitlab, type: :boolean, aliases: '-gl',
+                                                desc: 'Skip adding GitLab CI configuration'
+
+      class_option :ruby_version, type: :string, aliases: '-v',
+                                                 desc: "Tell us the ruby version which you use for the application. Default to Ruby #{DEFAULT_RUBY_VERSION}, which will cause the action to use the version specified in the #{RUBY_VERSION_FILE} file."
+
+      def add_brakeman_gem
```

Comment:
use gem method to add gems
https://api.rubyonrails.org/v7.1.3.2/classes/Rails/Generators/Actions.html#method-i-gem